docs/guides/tanzu-mission-control_tanzu_kubernetes_cluster.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: "Provisioning of a Tanzu Kubernetes Cluster"
+Description: |-
+    An example of provisioning Tanzu Kubernetes Grid Service (UTKG)
+---
+
+# Tanzu Kubernetes Cluster (UTKG)
```

Comment:
is it a good idea to use `UTKG` in the public doc? as it is internal term, you can think about using "Class-based Cluster".
Refs of TKG document: https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/2.4/using-tkg/config-ref.html#kubernetes-tuning-classbased-clusters-only-5